setup.py

```diff
@@ -58,6 +59,8 @@
         "webviz-config>=0.0.55",
         "webviz-subsurface-components>=0.0.23",
         "xtgeo>=2.8",
+        "opm>=2020.10.1",
```

Comment:
Unfortunately, `opm` PyPI binaries I think are not available for iOS and Windows as of now.
```suggestion
        "opm>=2020.10.1; sys_platform=='linux'",
```

Reply:
Good point. I will add the same requirement for the `ecl2df` package then.